getter/utils.go

```diff
@@ -0,0 +1,100 @@
+package getter
+
+import (
+	"bytes"
+	"compress/gzip"
+	_ "embed"
+	"encoding/csv"
+	"io"
+	"strconv"
+	"sync"
+
+	locodedb "github.com/nspcc-dev/locode-db/cmd/locode/db"
+)
+
+//go:embed data/countries.csv.gz
+var countriesData []byte
+
+//go:embed data/locodes.csv.gz
+var locodesData []byte
+
+var locodeDataOnce sync.Once
+
+func InitLocodeData() {
+	locodeDataOnce.Do(func() {
+		unpackCountriesData(&countriesData, countriesMap)
+		unpackLocodesData(&locodesData, locodesMap)
+	})
+}
+
+type LocodesCSV struct {
+	LocationName string
+	Continent    *locodedb.Continent
+	SubDivCode   string
+	SubDivName   string
+	Point        *locodedb.Point
+}
+
+var countriesMap = make(map[locodedb.CountryCode]string)
+var locodesMap = make(map[string]LocodesCSV)
+
+func unpackCountriesData(data *[]byte, m map[locodedb.CountryCode]string) {
```

Comment:
If you're passing a pointer to `data`, there needs to be a reason for it. For example, you can set the slice to `nil` after unpack (releasing the memory).

Reply:
But better have it like `unpackCountries(data []byte) map[locodedb.CountryCode]string` and manage package-level globals in the `InitLocodeData`.